MessagesView/MessagesView.swift

```diff
@@ -146,7 +150,8 @@ extension MessagesView : UICollectionViewDataSource {
             return footerView
             
         default:
```

Comment:
This have to be reverted into fatalError otherwise framework won't compile in Release mode, which is used by carthage by default
http://www.theboffinlab.com/lab-notes/swift-fatalerror-switch-statements